### ISSUE TITLE
How to represent dataset source?

### ISSUE THREAD

**Issue by simonff** [radiantearth/stac-spec#210]
For hosted datasets (eg, for MODIS MOD13Q1 dataset in the EE catalog) I want to point at the original file source. Should I use links for that? If yes, what do I put as 'rel' instead of XXX?

"links": [
        { "rel": "XXX", "href": " https://e4ftl01.cr.usgs.gov/MOLT/MOD13Q1.006/ " },
]

If no, should I create a separate Host section?

**Comment by m-mohr** [comment]
Isn't that similar to what we are discussing in #179? Sounds like this could be solved by the `derived_from` relation.

**Comment by simonff** [comment]
It is a very simple case of provenance. But while specifying provenance in general is very hard and not always needed, specifying the data source (especially in case of EE that is mostly a mirror) is very easy and always necessary.

**Comment by m-mohr** [comment]
I think my answer is the same as in #226, respectively #225.

**Comment by cholmes** [comment]
When I first started thinking about this I did think there'd be a 'derived_from' as well as 'copy_of' (or something like that). The first representing that there was processing done, the second just that it's stored in a different location - but that it points back to where it came from. As mentioned in #225 - I'd see 'copy_of' include the 'metadata processing' - unzipping, putting into a COG, etc.

I'm hesitant to add two more core link relationships, since derived_from seems like a stretch with few implementations and so we already have one in. I could see an extra attribute on derived_from that indicates that it is just a copy, not actual processing. Or just make a new link type, but we 'incubate' it for a bit. But I'd say if GEE uses it, and we also have Sentinel & Landsat in AWS also link back to their 'source' then it'd be pretty easy to bring in to the core.
